introduction/problems-solved.md

```diff
@@ -0,0 +1,40 @@
+This file holds the text to accompany the [What Problems Does InnerSource Solve?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321607) video.
```

Comment:
Suggest rewording intro graf to be less wordy: 

Inner source encourages and rewards collaboration and code reuse with anyone, regardless of their position in a company's organizational structure. This approach differs what is seen in traditional organizations where ideas and work product tend to stay trapped within the boundaries of the internal corporate hierarchy and its silos.

Reply:
I love reducing words!

Reply:
love this, @KathrynKLewis !